### ISSUE TITLE
WSL: winter/wn-system-module is locked to version v1.1.3 and an update of this package was not requested

### ISSUE THREAD

**Issue by softbia** [wintercms/winter#260]
- Winter CMS Build:1.1.3
- PHP Version: 7.4.3
- Composer Version: 2.1.5
- Ubuntu 20.04.2 LTS (on WSL2)

### Description:

When I'm trying to install WinterCMS 1.1.3 using composer, on a fresh install of Ubuntu on WSL2, I get 3 stated problems. Don't know how to solve those.

Full log:
```bash
$ composer create-project wintercms/winter wintercms
Creating a "wintercms/winter" project at "./wintercms"
Installing wintercms/winter (v1.1.3)
  - Installing wintercms/winter (v1.1.3): Extracting archive
Created project in /home/tobias/www/wintercms
Loading composer repositories with package information
Updating dependencies
Lock file operations: 106 installs, 0 updates, 0 removals
  - Locking composer/installers (dev-main 0a9a8dd)
  - Locking composer/semver (3.2.5)
  - Locking dms/phpunit-arraysubset-asserts (v0.2.1)
  - Locking doctrine/cache (2.1.1)
  - Locking doctrine/dbal (2.13.2)
  - Locking doctrine/deprecations (v0.5.3)
  - Locking doctrine/event-manager (1.1.1)
  - Locking doctrine/inflector (2.0.3)
  - Locking doctrine/instantiator (1.4.0)
  - Locking doctrine/lexer (1.2.1)
  - Locking dragonmantank/cron-expression (v2.3.1)
  - Locking egulias/email-validator (2.1.25)
  - Locking erusev/parsedown (1.7.4)
  - Locking erusev/parsedown-extra (0.8.1)
  - Locking fakerphp/faker (v1.15.0)
  - Locking hamcrest/hamcrest-php (v2.0.1)
  - Locking laravel/framework (v6.20.30)
  - Locking laravel/tinker (v2.6.1)
  - Locking league/commonmark (1.6.6)
  - Locking league/csv (9.7.1)
  - Locking league/flysystem (1.1.4)
  - Locking league/mime-type-detection (1.7.0)
  - Locking linkorb/jsmin-php (1.0.0)
  - Locking mockery/mockery (1.4.3)
  - Locking monolog/monolog (2.3.2)
  - Locking myclabs/deep-copy (1.10.2)
  - Locking nesbot/carbon (2.51.1)
  - Locking nikic/php-parser (v4.12.0)
  - Locking opis/closure (3.6.2)
  - Locking paragonie/random_compat (v9.99.99)
  - Locking phar-io/manifest (2.0.3)
  - Locking phar-io/version (3.1.0)
  - Locking php-parallel-lint/php-parallel-lint (v1.3.0)
  - Locking phpdocumentor/reflection-common (2.2.0)
  - Locking phpdocumentor/reflection-docblock (5.2.2)
  - Locking phpdocumentor/type-resolver (1.4.0)
  - Locking phpoption/phpoption (1.7.5)
  - Locking phpspec/prophecy (1.13.0)
  - Locking phpunit/php-code-coverage (9.2.6)
  - Locking phpunit/php-file-iterator (3.0.5)
  - Locking phpunit/php-invoker (3.1.1)
  - Locking phpunit/php-text-template (2.0.4)
  - Locking phpunit/php-timer (5.0.3)
  - Locking phpunit/phpunit (9.5.7)
  - Locking psr/container (1.1.1)
  - Locking psr/log (1.1.4)
  - Locking psr/simple-cache (1.0.1)
  - Locking psy/psysh (v0.10.8)
  - Locking ramsey/uuid (3.9.3)
  - Locking scssphp/scssphp (v1.6.0)
  - Locking sebastian/cli-parser (1.0.1)
  - Locking sebastian/code-unit (1.0.8)
  - Locking sebastian/code-unit-reverse-lookup (2.0.3)
  - Locking sebastian/comparator (4.0.6)
  - Locking sebastian/complexity (2.0.2)
  - Locking sebastian/diff (4.0.4)
  - Locking sebastian/environment (5.1.3)
  - Locking sebastian/exporter (4.0.3)
  - Locking sebastian/global-state (5.0.3)
  - Locking sebastian/lines-of-code (1.0.3)
  - Locking sebastian/object-enumerator (4.0.4)
  - Locking sebastian/object-reflector (2.0.4)
  - Locking sebastian/recursion-context (4.0.4)
  - Locking sebastian/resource-operations (3.0.3)
  - Locking sebastian/type (2.3.4)
  - Locking sebastian/version (3.0.2)
  - Locking squizlabs/php_codesniffer (3.6.0)
  - Locking swiftmailer/swiftmailer (v6.2.7)
  - Locking symfony/console (v4.4.29)
  - Locking symfony/css-selector (v5.3.4)
  - Locking symfony/debug (v4.4.27)
  - Locking symfony/deprecation-contracts (v2.4.0)
  - Locking symfony/error-handler (v4.4.27)
  - Locking symfony/event-dispatcher (v4.4.27)
  - Locking symfony/event-dispatcher-contracts (v1.1.9)
  - Locking symfony/finder (v4.4.27)
  - Locking symfony/http-client-contracts (v2.4.0)
  - Locking symfony/http-foundation (v4.4.29)
  - Locking symfony/http-kernel (v4.4.29)
  - Locking symfony/mime (v5.3.4)
  - Locking symfony/polyfill-ctype (v1.23.0)
  - Locking symfony/polyfill-iconv (v1.23.0)
  - Locking symfony/polyfill-intl-idn (v1.23.0)
  - Locking symfony/polyfill-intl-normalizer (v1.23.0)
  - Locking symfony/polyfill-mbstring (v1.23.1)
  - Locking symfony/polyfill-php72 (v1.23.0)
  - Locking symfony/polyfill-php73 (v1.23.0)
  - Locking symfony/polyfill-php80 (v1.23.1)
  - Locking symfony/process (v4.4.27)
  - Locking symfony/routing (v4.4.27)
  - Locking symfony/service-contracts (v2.4.0)
  - Locking symfony/translation (v4.4.27)
  - Locking symfony/translation-contracts (v2.4.0)
  - Locking symfony/var-dumper (v4.4.27)
  - Locking symfony/yaml (v3.4.47)
  - Locking theseer/tokenizer (1.2.1)
  - Locking tijsverkoyen/css-to-inline-styles (2.2.3)
  - Locking twig/twig (v2.14.6)
  - Locking vlucas/phpdotenv (v3.6.8)
  - Locking webmozart/assert (1.10.0)
  - Locking wikimedia/composer-merge-plugin (v2.0.1)
  - Locking wikimedia/less.php (v3.1.0)
  - Locking winter/storm (v1.1.3)
  - Locking winter/wn-backend-module (v1.1.3)
  - Locking winter/wn-cms-module (v1.1.3)
  - Locking winter/wn-system-module (v1.1.3)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 106 installs, 0 updates, 0 removals
  - Installing wikimedia/composer-merge-plugin (v2.0.1): Extracting archive
  - Installing composer/installers (dev-main 0a9a8dd): Extracting archive
  - Installing sebastian/version (3.0.2): Extracting archive
  - Installing sebastian/type (2.3.4): Extracting archive
  - Installing sebastian/resource-operations (3.0.3): Extracting archive
  - Installing sebastian/recursion-context (4.0.4): Extracting archive
  - Installing sebastian/object-reflector (2.0.4): Extracting archive
  - Installing sebastian/object-enumerator (4.0.4): Extracting archive
  - Installing sebastian/global-state (5.0.3): Extracting archive
  - Installing sebastian/exporter (4.0.3): Extracting archive
  - Installing sebastian/environment (5.1.3): Extracting archive
  - Installing sebastian/diff (4.0.4): Extracting archive
  - Installing sebastian/comparator (4.0.6): Extracting archive
  - Installing sebastian/code-unit (1.0.8): Extracting archive
  - Installing sebastian/cli-parser (1.0.1): Extracting archive
  - Installing phpunit/php-timer (5.0.3): Extracting archive
  - Installing phpunit/php-text-template (2.0.4): Extracting archive
  - Installing phpunit/php-invoker (3.1.1): Extracting archive
  - Installing phpunit/php-file-iterator (3.0.5): Extracting archive
  - Installing theseer/tokenizer (1.2.1): Extracting archive
  - Installing nikic/php-parser (v4.12.0): Extracting archive
  - Installing sebastian/lines-of-code (1.0.3): Extracting archive
  - Installing sebastian/complexity (2.0.2): Extracting archive
  - Installing sebastian/code-unit-reverse-lookup (2.0.3): Extracting archive
  - Installing phpunit/php-code-coverage (9.2.6): Extracting archive
  - Installing symfony/polyfill-ctype (v1.23.0): Extracting archive
  - Installing webmozart/assert (1.10.0): Extracting archive
  - Installing phpdocumentor/reflection-common (2.2.0): Extracting archive
  - Installing phpdocumentor/type-resolver (1.4.0): Extracting archive
  - Installing phpdocumentor/reflection-docblock (5.2.2): Extracting archive
  - Installing doctrine/instantiator (1.4.0): Extracting archive
  - Installing phpspec/prophecy (1.13.0): Extracting archive
  - Installing phar-io/version (3.1.0): Extracting archive
  - Installing phar-io/manifest (2.0.3): Extracting archive
  - Installing myclabs/deep-copy (1.10.2): Extracting archive
  - Installing phpunit/phpunit (9.5.7): Extracting archive
  - Installing dms/phpunit-arraysubset-asserts (v0.2.1): Extracting archive
  - Installing doctrine/cache (2.1.1): Extracting archive
  - Installing doctrine/deprecations (v0.5.3): Extracting archive
  - Installing doctrine/event-manager (1.1.1): Extracting archive
  - Installing doctrine/inflector (2.0.3): Extracting archive
  - Installing doctrine/lexer (1.2.1): Extracting archive
  - Installing dragonmantank/cron-expression (v2.3.1): Extracting archive
  - Installing erusev/parsedown (1.7.4): Extracting archive
  - Installing symfony/deprecation-contracts (v2.4.0): Extracting archive
  - Installing psr/container (1.1.1): Extracting archive
  - Installing fakerphp/faker (v1.15.0): Extracting archive
  - Installing league/commonmark (1.6.6): Extracting archive
  - Installing league/mime-type-detection (1.7.0): Extracting archive
  - Installing league/flysystem (1.1.4): Extracting archive
  - Installing hamcrest/hamcrest-php (v2.0.1): Extracting archive
  - Installing mockery/mockery (1.4.3): Extracting archive
  - Installing psr/log (1.1.4): Extracting archive
  - Installing monolog/monolog (2.3.2): Extracting archive
  - Installing opis/closure (3.6.2): Extracting archive
  - Installing php-parallel-lint/php-parallel-lint (v1.3.0): Extracting archive
  - Installing psr/simple-cache (1.0.1): Extracting archive
  - Installing symfony/polyfill-php80 (v1.23.1): Extracting archive
  - Installing symfony/polyfill-php72 (v1.23.0): Extracting archive
  - Installing symfony/polyfill-mbstring (v1.23.1): Extracting archive
  - Installing symfony/var-dumper (v4.4.27): Extracting archive
  - Installing symfony/service-contracts (v2.4.0): Extracting archive
  - Installing symfony/polyfill-php73 (v1.23.0): Extracting archive
  - Installing symfony/console (v4.4.29): Extracting archive
  - Installing psy/psysh (v0.10.8): Extracting archive
  - Installing paragonie/random_compat (v9.99.99): Extracting archive
  - Installing ramsey/uuid (3.9.3): Extracting archive
  - Installing squizlabs/php_codesniffer (3.6.0): Extracting archive
  - Installing symfony/polyfill-intl-normalizer (v1.23.0): Extracting archive
  - Installing symfony/polyfill-intl-idn (v1.23.0): Extracting archive
  - Installing symfony/polyfill-iconv (v1.23.0): Extracting archive
  - Installing egulias/email-validator (2.1.25): Extracting archive
  - Installing swiftmailer/swiftmailer (v6.2.7): Extracting archive
  - Installing symfony/debug (v4.4.27): Extracting archive
  - Installing symfony/event-dispatcher-contracts (v1.1.9): Extracting archive
  - Installing symfony/finder (v4.4.27): Extracting archive
  - Installing symfony/mime (v5.3.4): Extracting archive
  - Installing symfony/http-foundation (v4.4.29): Extracting archive
  - Installing symfony/http-client-contracts (v2.4.0): Extracting archive
  - Installing symfony/event-dispatcher (v4.4.27): Extracting archive
  - Installing symfony/error-handler (v4.4.27): Extracting archive
  - Installing symfony/http-kernel (v4.4.29): Extracting archive
  - Installing symfony/process (v4.4.27): Extracting archive
  - Installing symfony/routing (v4.4.27): Extracting archive
  - Installing symfony/translation-contracts (v2.4.0): Extracting archive
  - Installing symfony/translation (v4.4.27): Extracting archive
  - Installing symfony/css-selector (v5.3.4): Extracting archive
  - Installing tijsverkoyen/css-to-inline-styles (2.2.3): Extracting archive
  - Installing phpoption/phpoption (1.7.5): Extracting archive
  - Installing vlucas/phpdotenv (v3.6.8): Extracting archive
  - Installing wikimedia/less.php (v3.1.0): Extracting archive
  - Installing twig/twig (v2.14.6): Extracting archive
  - Installing symfony/yaml (v3.4.47): Extracting archive
  - Installing scssphp/scssphp (v1.6.0): Extracting archive
  - Installing nesbot/carbon (2.51.1): Extracting archive
  - Installing linkorb/jsmin-php (1.0.0): Extracting archive
  - Installing league/csv (9.7.1): Extracting archive
  - Installing laravel/framework (v6.20.30): Extracting archive
  - Installing laravel/tinker (v2.6.1): Extracting archive
  - Installing erusev/parsedown-extra (0.8.1): Extracting archive
  - Installing doctrine/dbal (2.13.2): Extracting archive
  - Installing winter/storm (v1.1.3): Extracting archive
  - Installing winter/wn-backend-module (v1.1.3): Extracting archive
  - Installing winter/wn-cms-module (v1.1.3): Extracting archive
  - Installing composer/semver (3.2.5): Extracting archive
  - Installing winter/wn-system-module (v1.1.3): Extracting archive
75 package suggestions were added by new dependencies, use `composer suggest` to see details.
Package sebastian/resource-operations is abandoned, you should avoid using it. No replacement was suggested.
Generating autoload files
75 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

Running composer update to apply merge settings
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - winter/wn-system-module is locked to version v1.1.3 and an update of this package was not requested.
    - winter/wn-system-module v1.1.3 requires composer/installers dev-main as 1.0 -> found composer/installers[dev-main, 2.x-dev (alias of dev-main)] but it conflicts with your root composer.json require (~1.0).
  Problem 2
    - winter/wn-backend-module is locked to version v1.1.3 and an update of this package was not requested.
    - winter/wn-backend-module v1.1.3 requires composer/installers dev-main as 1.0 -> found composer/installers[dev-main, 2.x-dev (alias of dev-main)] but it conflicts with your root composer.json require (~1.0).
  Problem 3
    - winter/wn-cms-module is locked to version v1.1.3 and an update of this package was not requested.
    - winter/wn-cms-module v1.1.3 requires composer/installers dev-main as 1.0 -> found composer/installers[dev-main, 2.x-dev (alias of dev-main)] but it conflicts with your root composer.json require (~1.0).

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Update to apply merge settings failed, reverting ./composer.lock to its original content.
> php artisan winter:version
*** Detecting Winter CMS build...
*** No database detected - skipping setting the build number.
*** Detected Winter CMS build 1.1.3.
> php artisan package:discover
Discovered Package: laravel/tinker
Discovered Package: nesbot/carbon
Package manifest generated successfully.
> php artisan key:generate
Application key [base64:uxnmkqcN22057F1b9SImEU8w7GE2bpxJOgzRk+OJsNw=] set successfully.
> php artisan package:discover
Discovered Package: laravel/tinker
Discovered Package: nesbot/carbon
Package manifest generated successfully.
```

Distro description:
```bash
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.2 LTS
Release:        20.04
Codename:       focal
```

### Steps To Reproduce:

1. Fresh install of Ubuntu WSL2 on Windows
2. Followed the instructions on the website for [minimum requirements](https://wintercms.com/docs/setup/installation#system-requirements):  `sudo apt-get update && sudo apt-get install php php-ctype php-curl php-xml php-fileinfo php-gd php-json php-mbstring php-mysql php-sqlite3 php-zip`
3. [Installed composer on WSL2](https://getcomposer.org/download/)
4. Ran [this command](https://wintercms.com/docs/console/commands#console-install-composer) inside the WSL2 distro: `composer create-project wintercms/winter wintercms`

**Comment by bennothommo** [comment]
@softbia Does it work if you run another `composer update`, or do the same problems appear?

**Comment by LukeTowers** [comment]
This is a known issue that will be fixed in 1.1.4, you can use dev-develop for now. Update hopefully out tomorrow.
